.gitignore

```diff
@@ -2,3 +2,4 @@
 .*.swp
 coverage/
 .nyc_output/
+.idea
```

Comment:
Generally this belongs in a global git config, rather than polluting specific git repos with IDE-specific knowledge (nbd tho)